### PR TITLE
Add Kagi theme

### PR DESCRIPTION
**Pull request by tyler-dot-earth**
I use the privacy-focused search engine [kagi](https://kagi.com) and which provides [css customization](https://kagi.com/settings?p=custom_css). I am in no way affiliated with them beyond using the service.

This change adds `kagi/kagi.css` with some simple Everforest theming applied.

Screenshots:

image

image

## Summary by Sourcery

Add a new Everforest theme for the Kagi search engine, including a CSS file and documentation on how to use it.

New Features:
- Introduce a new CSS theme for the Kagi search engine, applying Everforest theming.

Documentation:
- Add a README file explaining the Everforest theming for Kagi and instructions on how to apply it.

### CODE REVIEW

**Comment by tyler-dot-earth** [comment]
@sourcery-ai review

File: kagi/README.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+--dd-list-input-bg: var(--input-bg);
+--not-found-bubble-bg: #f2f2f2;
+--app-sidebar-item-border: rgba(25,25,25,.2);
+--app-sidebar-link: rgba(25,25,25,.6);
+--app-sidebar-nav-item-bg_hover: #ffb319;
+--app-sidebar-nav-item-icon_hover: #191919;
+--ranked-box-shadow: rgba(0,0,0,.25);
+--ranked-box-connection-secure: #529348;
+--ranked-box-connection-insecure: #da5819;
+--ranked-box-tracker-desc-danger: #da5819;
```
[the remark this answers is not in the document]
Comment (tyler-dot-earth) [reply kagi/README.md:141]: It is defined twice like that if you view the variables on the homepage, so leaving as-is.

File: kagi/kagi.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	--landing-bg: var(--ef-bg-0);
+	--app-frame-bg: var(--ef-bg-0);
+	--app-bg: var(--ef-bg-1);
+
+	--color-search-input: var(--ef-bg-1);
+	--color-search-input-border: var(--ef-bg-2);
+
+	--inline-widget-bg: var(--ef-bg-2);
+
+	--search-result-title: var(--ef-gray-2);
```
[the remark this answers is not in the document]
Comment (tyler-dot-earth) [reply kagi/kagi.css:217]: whoops, wrong version when extracting to file.

**Comment by tyler-dot-earth** [comment]
@sourcery-ai review

**Comment by neuromaancer** [comment]
Thanks!

**Pull request merged**